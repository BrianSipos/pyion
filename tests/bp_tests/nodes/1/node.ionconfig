# ==================================================================================================== node.ionconfig ====================================================================================================
# Auto-generated @ Tuesday, 12. February 2019 09:30AM
# ========================================================================================================================================================================================================================

# Name of the volatile memory
wmKey 65281

# Size of the volatile memory in bytes
wmSize 50000000

# Name of the non-volatile memory
sdrName ion

# Size of the non-volatile memory in words (e.g. 32-bit for 32-bit computers)
#heapWords 7812500
heapWords 50000

# Config flags for SDR: 1=SDR in RAM, 2=SDR in file (for others see ION manual)
configFlags 1

# If SDR is file, then directory to store it
pathName /tmp

# ========================================================================================================= EOF =========================================================================================================