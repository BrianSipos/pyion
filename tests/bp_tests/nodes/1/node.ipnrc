# ==================================================================================================== node.ipnrc ====================================================================================================
# Auto-generated @ Tuesday, 12. February 2019 09:30AM
# ====================================================================================================================================================================================================================

# Define a plan (possibility to tx) to node node2 using outduct udp
a plan 2 udp/196.128.0.102:4602

# ======================================================================================================= EOF =======================================================================================================