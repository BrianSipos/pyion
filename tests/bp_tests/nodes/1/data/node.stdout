./start: line 42: ./ionstart: No such file or directory
